### ISSUE TITLE
[Dec 2025] Accessibility Audit: Org sign-up flow

### ISSUE THREAD

**Issue by DanielSass** [CDCgov/prime-simplereport#9344]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the org sign-up user flow. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/sign-up`
* `/app/sign-up/identity-verification`

Use cases to test:
* user selects that their org is already using SimpleReport
* user selects that they are trying to view their results
* user signs up for SimpleReport but is in a state we don't support yet and are prompted to join the waitlist
* user goes through identity verification workflow
    * they pass the Experian check
    * they are prompted to call to validate their identity
* after org identity verified, cont user sign up (inc all forms of MFA)

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.
Export scan results as a csv and add the file to this ticket.

### Additional context
See tickets from previous audits  #8521 #5765, #6325
Experian answers for ID verification step: 1, 4, 2, 1, 3 (1 is first option)

**Comment by mpbrown** [comment]
Findings:

- Sign up - select status [dev5.simplereport.gov-Sign-up-select-status-2026-01-12.json](https://github.com/user-attachments/files/24680821/dev5.simplereport.gov-Sign-up-select-status-2026-01-12.json)
- Request access [dev5.simplereport.gov-Request-access-2026-01-12.json](https://github.com/user-attachments/files/24680840/dev5.simplereport.gov-Request-access-2026-01-12.json)
- Get COVID-19 test results [dev5.simplereport.gov-Get-results-2026-01-12.json](https://github.com/user-attachments/files/24680870/dev5.simplereport.gov-Get-results-2026-01-12.json)
- Sign up - organization information [dev5.simplereport.gov-Sign-up-organization-information-2026-01-13.csv](https://github.com/user-attachments/files/24680888/dev5.simplereport.gov-Sign-up-organization-information-2026-01-13.csv)
- Sign up - identity verification consent [dev5.simplereport.gov-Sign-up-identity-verification-consent-2026-01-12.json](https://github.com/user-attachments/files/24680898/dev5.simplereport.gov-Sign-up-identity-verification-consent-2026-01-12.json)
- Sign up - personal details [dev5.simplereport.gov-Sign-up-personal-details-2026-01-13.json](https://github.com/user-attachments/files/24680900/dev5.simplereport.gov-Sign-up-personal-details-2026-01-13.json)
- Sign up - identity verification [localhost_3000-Sign-up-identity-verification-2026-01-16.csv](https://github.com/user-attachments/files/24680909/localhost_3000-Sign-up-identity-verification-2026-01-16.csv)
- Sign up - success (**this page is no longer available to users now that we always default to having the Experian verification failing**)
- Create your password [test.simplereport.gov-Create-your-password-2026-01-13.json](https://github.com/user-attachments/files/24680954/test.simplereport.gov-Create-your-password-2026-01-13.json)
- Session timeout [localhost_3000-Session-timeout-or-SimpleReport-2026-01-16.json](https://github.com/user-attachments/files/24680986/localhost_3000-Session-timeout-or-SimpleReport-2026-01-16.json)
- Account set up - Step 2 - Security question [test.simplereport.gov-Select-your-security-question-2026-01-13.json](https://github.com/user-attachments/files/24680995/test.simplereport.gov-Select-your-security-question-2026-01-13.json)
- Account set up - Step 3 - MFA complete [localhost_6006-Account-set-up-Step-3c_-MFA-complete-2026-01-16.json](https://github.com/user-attachments/files/24681016/localhost_6006-Account-set-up-Step-3c_-MFA-complete-2026-01-16.json)
- Account set up - Step 3b - Verify Email secuirty code [localhost_6006-Account-set-up-Step-3b_-Verify-Email-security-cod-2026-01-16.json](https://github.com/user-attachments/files/24681019/localhost_6006-Account-set-up-Step-3b_-Verify-Email-security-cod-2026-01-16.json)
- Account set up - Step 3a - Google Authenticator MFA [localhost_6006-Account-set-up-Step-3a_-Google-Authenticator-MFA-2026-01-16.json](https://github.com/user-attachments/files/24681023/localhost_6006-Account-set-up-Step-3a_-Google-Authenticator-MFA-2026-01-16.json)
- Account set up - Step 3b - Verify Google Authenticator [localhost_6006-Account-set-up-Step-3b_-Verify-Google-Authenticat-2026-01-16.json](https://github.com/user-attachments/files/24681026/localhost_6006-Account-set-up-Step-3b_-Verify-Google-Authenticat-2026-01-16.json)
- Account set up - Step 3a - Okta Verify MFA [localhost_6006-Account-set-up-Step-3a_-Okta-Verify-MFA-2026-01-16.json](https://github.com/user-attachments/files/24681027/localhost_6006-Account-set-up-Step-3a_-Okta-Verify-MFA-2026-01-16.json)
- Account set up - Step 3b - Verify Okta Verify security code [localhost_6006-Account-set-up-Step-3b_-Verify-Okta-Verify-securi-2026-01-16.json](https://github.com/user-attachments/files/24681028/localhost_6006-Account-set-up-Step-3b_-Verify-Okta-Verify-securi-2026-01-16.json)
- Account set up - Step 3a - Voice Call MFA [localhost_6006-Account-set-up-Step-3a_-Voice-Call-MFA-2026-01-16.json](https://github.com/user-attachments/files/24681032/localhost_6006-Account-set-up-Step-3a_-Voice-Call-MFA-2026-01-16.json)
- Account set up - Step 3b - Verify Voice Call security code [localhost_6006-Account-set-up-Step-3b_-Verify-Voice-Call-securit-2026-01-16.json](https://github.com/user-attachments/files/24681033/localhost_6006-Account-set-up-Step-3b_-Verify-Voice-Call-securit-2026-01-16.json)
- Account set up - Step 3a - Security key [localhost_6006-Account-set-up-Step-3a_-Security-key-2026-01-16.json](https://github.com/user-attachments/files/24681036/localhost_6006-Account-set-up-Step-3a_-Security-key-2026-01-16.json)
- Account set up - Step 3 - MFA Select [test.simplereport.gov-Set-up-authentication-2026-01-13.json](https://github.com/user-attachments/files/24681045/test.simplereport.gov-Set-up-authentication-2026-01-13.json)
- Account set up - Step 3a - SMS MFA [test.simplereport.gov-Set-up-authentication-via-text-message-(SM-2026-01-13.json](https://github.com/user-attachments/files/24681047/test.simplereport.gov-Set-up-authentication-via-text-message-.SM-2026-01-13.json)
- Account set up - Step 3b - Verify SMS security code [test.simplereport.gov-Verify-your-security-code-from-text-messag-2026-01-13.json](https://github.com/user-attachments/files/24681048/test.simplereport.gov-Verify-your-security-code-from-text-messag-2026-01-13.json)

**Comment by mpbrown** [comment]
Issues created:

- https://github.com/CDCgov/prime-simplereport/issues/9388

This particular issue would likely become obsolete if we update the app to remove the restrictions on organizations signing up from unsupported STLTs since now that we're sending to AIMS, all STLTs should be supported.
